src/main/java/net/william278/velocitab/config/Settings.java

```diff
@@ -21,10 +21,12 @@
 public class Settings {
 
     @YamlKey("headers")
-    private Map<String, String> headers = Map.of("default", "&rainbow&Running Velocitab by William278");
+    @YamlComment("List of Headers to rotate through.")
```

Comment:
```suggestion
    @YamlComment("Header(s) to display above the TAB list for each server group.\nList multiple headers and set update_rate to the number of ticks between frames for basic animations")
```
Would be nice to just slightly improve the comment clarity